src/schibsted/payment/lib/Resource.php

```diff
@@ -16,6 +16,7 @@
     protected $_sdk = null;
 
     protected $_sdk_class = '';
+    protected $name;
```

Comment:
$name field has to be, and is, set on the sub classes. Abstract field, if there was such a thing.

Reply:
This was mostly for IDE complaints :-)